File: app/src/main/java/com/inspiringteam/xchange/data/source/remote/QuakesApiService.java
package com.inspiringteam.xchange.data.source.remote;

import com.inspiringteam.xchange.data.models.QuakesResponse;


import io.reactivex.Observable;
import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Query;


public interface QuakesApiService {
    @GET("1.0_day.geojson")
    Single<QuakesResponse> getQuakes();
}

File: app/src/main/java/com/inspiringteam/xchange/data/source/remote/QuakesApiService.kt
package com.inspiringteam.xchange.data.source.remote

import retrofit2.http.GET
import com.inspiringteam.xchange.data.models.QuakesResponse
import io.reactivex.Single

interface QuakesApiService {
    @get:GET("1.0_day.geojson")
    val quakes: Single<QuakesResponse>
}